web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -77,6 +77,7 @@ export type IntegrationTemplate<
 // resource's subKind field.
 export enum IntegrationKind {
   AwsOidc = 'aws-oidc',
+  AWSRa = 'aws-ra',
```

Comment:
with comment b/c i don't think a lot would know what ra means
```suggestion
 /* AWS Roles Anywhere */
  AwsRa = 'aws-ra',
```